CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+#### Version 1.17.0 (2018-05-30)
+
+New Features:
+ - Add support for gzip compression levels
+   ([#1044](https://github.com/Shopify/sarama/pull/1044)).
+ - Add DeleteRecords operation to Broker
+   ([#1027](https://github.com/Shopify/sarama/pull/1027)).
+ - Add support for Metadata Request/Response
+   ([#1047](https://github.com/Shopify/sarama/pull/1047)).
+ - Add CreatePartitions to Broker
+   ([#1065](https://github.com/Shopify/sarama/pull/1065)).
+ - Add support for DeleteGroups
+   ([#1096](https://github.com/Shopify/sarama/pull/1096)).
+
+Improvements:
+ - Refactor ConsumerMetadataReq/Resp as part of `KIP-98`
```

Comment:
the relevant change for users is that ConsumerMetadataReq/Resp has been migrated to FindCoordinatorReq/Resp